omero/sysadmins/unix/install-web/install-apache.txt

```diff
@@ -8,12 +8,21 @@ Apache 2.2+ with mod_wsgi configuration (Unix/Linux)
 
 .. note:: Since OMERO 5.2, the OMERO.web framework no longer bundles
       a copy of the Django package, instead manual installation of
-      the Django dependency is required. It is highly recommended to use
+      the Django dependency is required. It is highly recomanded to use
       `Django 1.8`_ (LTS) which requires Python 2.7. For more information
       see :ref:`python-requirements` on the
       :doc:`/sysadmins/version-requirements` page.
 
-If you have installed Apache, install `mod\_wsgi <http://www.modwsgi.org/>`_.
+OMERO.web uses
+`Web Server Gateway Interface (WSGI) <http://wsgi.readthedocs.org/en/latest/learn.html>`_
+as a primary deployment platform, the Python standard
+`PEP 3333 <https://www.python.org/dev/peps/pep-3333/>`_
+that describes a web server communicates with web applications.
```

Comment:
should have a 'how' in that sentence I think:
"…describes how a web server communicates with web applications." ?